crypto/rand/rand_lib.c

```diff
@@ -25,6 +25,11 @@
 /* Macro to convert two thirty two bit values into a sixty four bit one */
 #define TWO32TO64(a, b) ((((uint64_t)(a)) << 32) + (b))
 
+/* Some libraries pretend to be glibc but don't support version checks */
```

Comment:
I find this comment misleading. Even though some libraries might pretend to be glibc (really? which one?), majority don't pretend anything.

For reference, what happens is simply that standard pre-processor doesn't handle undefined *parameterized* macros, period. Even in `defined(FOO) && FOO(what,ever)`. Well, one can resolve this with disjoint if, i.e.
```
#ifdef FOO
# if FOO(what,ever)
# endif
#endif
```
but single expression won't work. So that attempt to use __GLIBC_PREREQ(X,Y) is doomed to fail with *any* other libc.